guides/hack/21-XHP/07-basic-usage.md

```diff
@@ -154,6 +154,8 @@ is rendered (`toStringAsync()` is called) with any required attributes missing.
 With `check_xhp_attribute=true` (available since HHVM 4.8) this is also a
 typechecker error.
 
+Use the [`->:` operator](/hack/expressions-and-operators/XHP-attribute-selection) to select an attribute, or `$this->:` in place of the named class instance.
```

Comment:
Ditto, why is `$this->:` getting a special mention here?

Reply:
Removed.